### DISCUSSION TITLE
Is it possible to add a dwell time at start of each cut path element?

### DISCUSSION THREAD

**Discussion by frmdstryr** [meerk40t/meerk40t#2442] [Q&A] [answered]
I have access to a galvo fiber laser and am trying to cut stencils out of very thin steel sheet metal (5 thou or ~20 micron). 

The designs consist of many small rounded rectangles that it needs to cut out. It somewhat works but often it does not fully cut the rectangle and instead leaves a small "tab" at the start/stop point that must be manually broken off (which bends and damages the stencil). 

Is there any option or way to set a "dwell" time at the start of each path so the laser says longer there?

P.S. Thank you for making this, it's was really easy to use and works well for everything else I've used it for so far!

**Comment by tatarize** [comment, marked as answer]
Yes. You're referring to the laser-on delay.

![image](https://github.com/meerk40t/meerk40t/assets/3302478/2e6a1138-9be6-4704-87b4-056634d757e3)


You probably want to just up that value in global, since if it's not turning on long enough and missing the start of the cut, you'd want to increase that. You can do that for each operations if you want a operation specific version.

![image](https://github.com/meerk40t/meerk40t/assets/3302478/60c04b92-d226-4138-9d10-20e6be23e4dc)

The functionality is built into the laser. Sometimes the mirrors kick in and start moving too early so you get a little bit of movement before the laser gets up to power, and if that's the case you want to give the laser some added time. Though you sort of need to figure out the right amount of time to use. 

https://youtu.be/gFvbrNnvijo?si=qMjrysqog5kc6C9l&t=243

Ezcad called these settings Start TC. Start Time Control, though most other programs and devices generally call this laser on delay which is how meerk40t has it set.
